src/transaction/log_tran_table.c

```diff
@@ -1683,10 +1683,11 @@ logtb_finalize_tdes (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
 {
   int r;
 
-  // *INDENT-OFF*
-  tdes->client.~clientids ();
-  tdes->m_modified_classes.~tx_transient_class_registry ();
-  // *INDENT-ON*
+  /*
```

Comment:
Do you want to keep this commented?

Reply:
It look better to remove the comment. :)